src/signTx.c

```diff
@@ -1837,7 +1923,8 @@ void signTx_handleAPDU(
 	case SIGN_STAGE_BODY_WITHDRAWALS:
 	case SIGN_STAGE_BODY_VALIDITY_INTERVAL:
 	case SIGN_STAGE_BODY_MINT:
-	case SIGN_STAGE_BODY_MINT_SUBMACHINE: {
+	case SIGN_STAGE_BODY_MINT_SUBMACHINE:
+	case SIGN_STAGE_BODY_SCRIPT_DATA_HASH: {
 		explicit_bzero(&BODY_CTX->stageData, SIZEOF(BODY_CTX->stageData));
 		break;
 	}
```

Comment:
Above this, there is a TODO I don't understand --- it seems to me that we still need to zero stageData in between the stages.

Reply:
Well, it's not my TODO so I'm not sure what the writer thought, but I agree, the way issue #66 was solved we still need to zero out the stageData.

Reply:
Removed the TODO.